models/object_detection_yolox/demo.cpp

```diff
@@ -306,6 +306,10 @@ int main(int argc, char** argv)
         tm.stop();
         cout << "Inference time: " << tm.getTimeMilli() << " ms\n";
         Mat img = visualize(predictions, frame, letterboxScale, tm.getFPS());
+        if (save)
```

Comment:
It is not good saving as `.jpg` when the input is camera stream. It just keeps overwriting the last one.

Reply:
Agreed. I've updated it to only save an image if the user has an input image.